ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -738,6 +738,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-upi-disconnected-sdn-workers-rhcos-rhel8
+- as: aws-ipi-hypershift-install-test-p1
+  cron: '@daily'
```

Comment:
@LiangquanLi930 as my last comment mentioned, please do not use `cron: '@daily'` because it depends on the mreged time. we should control it example, so, please take `0 1 * * *`. of course, you could choose other times.